src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form.html.twig

```diff
@@ -23,31 +23,66 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
 {% form_theme apiAccessForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
 
-{% block api_access_form_block %}
-  {{ form_start(apiAccessForm) }}
+{% block session_alert %}
+  {{ parent() }}
+
+  {# Display generated client secrets passed ny session #}
+  {% if app.session.flashbag.peek('client_secret')|length() > 0 %}
+    {% for clientSecret in app.session.flashbag.get('client_secret') %}
+      <div class="alert alert-success d-print-none" role="alert">
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+          <span aria-hidden="true"><i class="material-icons">close</i></span>
+        </button>
+        <div class="alert-text">
```

Comment:
I find it a little weird you have to re-implement the full Twig logic to display flash messages just for this. Looks a little overkill. Can you confirm there is no lighter way? No reusable Twig widget?

Reply:
Actually Nicolas implementation was based on the flash messages we already have, so there was no need to add this extra flash message container The problem is that it implied created a big text in HTML on the Controller side with HTML tags mixed into the wording which I didn't find very elegant

So I replaced it with this custom flash message with its dedicated bag